user-group/survey01-reminder.html

```diff
@@ -0,0 +1,107 @@
+                                        </td>
+                                    </tr>
+                                </tbody>
+                            </table>
+
+                            <table width="600" style="border-spacing:0;Margin:0 auto;">
+                                <tbody>
+                                    <tr>
+                                        <td style="padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;vertical-align:top;background-color:#eaecec;width:1px;">​</td>
+                                        <td width="600" bgcolor="#ffffff" style="padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;vertical-align:top;Margin:0 auto;">
+                                            <table width="100%" style="border-spacing:0;">
+                                                <tbody>
+                                                    <tr>
+                                                        <td style="vertical-align:top;padding-bottom:10px;padding-left:60px;padding-right:60px;padding-top:20px;">
+                                                            <table width="100%" style="border-spacing:0;">
+                                                                <tbody>
+                                                                    <tr>
+                                                                        <td style="padding-top:10px;padding-bottom:0;padding-right:0;padding-left:0;vertical-align:top;text-align:center">
+                                                                            <p style="Margin-top:0;font-weight:normal;color:#262626;font-family:sans-serif;font-size:16px;line-height:22px;Margin-bottom:15px;">The results of this survey will help inform how Ally can align with existing resources and processes and how it can most effectively allow instructors and students to reach out for help.</p>
+                                                                        </td>
+                                                                    </tr>
+                                                                </tbody>
+                                                            </table>
+                                                        </td>
+                                                    </tr>
+                                                </tbody>
+                                            </table>
+                                        </td>
+                                    </tr>
+                                </tbody>
+                            </table>
+
+                            <table width="600" style="border-spacing:0;Margin:0 auto;">
+                                <tbody>
+                                    <tr>
+                                        <td style="padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;vertical-align:top;background-color:#eaecec;width:1px;">​</td>
```

Comment:
As suggested above, this might be improved with `role="presentation"` if we split it into its own table.